types/koa/tsconfig.json

```diff
@@ -21,6 +21,11 @@
             "../"
         ],
         "types": [],
+        "paths": {
+            "http-errors": [
+                "http-errors/v1"
+            ]
+        },
```

Comment:
IMO, instead of moving the old version to `v1/`, you should create a `package.json` with the following contents:
```
{
    "private": true,
    "dependencies": {
        "@types/http-errors": "^1"
    }
}
```

You should then open a PR on the https://github.com/microsoft/DefinitelyTyped-tools repo and add the `@types/http-errors` to the `packages/definitions-parser/allowedPackageJsonDependencies.txt` file.

As far as I understand it, moving old packages into sub-directories is only meant for package versions that are still maintained by their authors and I don't think it's the case here.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58184#issuecomment-1087335019

Nevertheless, you should fix this...